src/nwb/NWBFile.hpp

```diff
@@ -127,9 +148,11 @@ class NWBFile
       const std::array<std::pair<std::string_view, std::string_view>, N>&
           specVariables);
 
+  std::unique_ptr<ElectrodeTable> elecTable;
   const std::string identifierText;
   std::shared_ptr<BaseIO> io;
   static std::vector<SizeType> emptyContainerIndexes;
+  inline const static std::string acquisitionPath = "/acquisition/";
```

Comment:
```suggestion
  inline const static std::string acquisitionPath = "/acquisition";
```

If I remember correctly, in other places, paths are defined without the "/" at the end. It would be useful to be consistent to make it easier to merge paths without having to check whether we already have "/" at the end.  We could consider switching to using ``std::filesystem::path`` for path strings (we just need to remember to call ``generic_string()`` to construct the path to avoid having "\" instead of "/" on Windows, because HDF5 I believe always wants paths with "/")

Reply:
Since HDF5 always expects "/" for paths and I don't think there's anywhere we're doing any path manipulation, I don't know if it's worth it to switch to `std::filesystem::path` right now. But I could see maybe it would become more useful to have the path objects when you're trying to determine the contents of groups and the file?